src/test/compile-fail/private-in-public-warn.rs

```diff
@@ -246,6 +245,8 @@ mod aliases_priv {
     use self::Priv1 as PrivUseAlias;
     use self::PrivTr1 as PrivUseAliasTr;
     type PrivAlias = Priv2;
+    //~^ WARN private type in public interface
+    //~| WARNING hard error
```

Comment:
>This seems like it's causing a new error?

No, the error from line 211 (alias location) migrated to this line (aliased type location).